src/pch.h

```diff
@@ -0,0 +1,75 @@
+#ifndef PCH_H
+#define PCH_H
+#pragma once
+
+/*
+    Only put files that never change or are rarely changed in this file.
+    An include graph will be provided for some headers that were not explicit
+    added. VC++ is a requirement.
+
+    We don't care about warnings for library files.
+*/
+#pragma warning(push)
+#pragma warning(disable : 4191 4365 4464 4571 4619 4623 4625 4626 4768 4774 5026 5027 5039 5045)
```

Comment:
`/Wall `is currently off but these are all the warnings the build would have reported with it on.